demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -582,6 +582,7 @@ def _is_price_changed(self) -> bool:
 
     def are_valid_files(self, id_set_validations) -> str:
         """Main Execution Method"""
+        self._is_pack_file_exists(self.pack_meta_file)
```

Comment:
Have you tested it? it will continue to other checks anyway this way and crash, shouldn't it ?

Reply:
Yes. see the new mechanism. It works but I'd rather use an exception instead, to ensure validation halts (currently, it's up to the developers maintaining SDK to prevent other tests from running). wdyt?